src/Conf.ml

```diff
@@ -1737,14 +1739,15 @@ let kind_of fname =
   | _ -> !kind
 
 let xdg_config =
-  match Caml.Sys.getenv_opt "XDG_CONFIG_HOME" with
-  | Some xdg_config_home ->
-      let filename =
-        Fpath.(
-          append (v xdg_config_home) (v "ocamlformat" / dot_ocamlformat))
-      in
-      if Fpath.exists filename then Some filename else None
-  | None -> None
+  let xdg_config_home =
+    match Caml.Sys.getenv_opt "XDG_CONFIG_HOME" with
+    | None | Some "" ->
+        let home = Option.value ~default:"" (Caml.Sys.getenv_opt "HOME") in
+        Fpath.(v home / ".config")
```

Comment:
If `HOME` is unset, `home` is the empty string and this raises `Invalid_argument`. I think returning `None` is this case is ok.